### DISCUSSION TITLE
how to clone for hindi language

### DISCUSSION THREAD

**Discussion by saichand018** [serp-ai/bark-with-voice-clone#96] [General]
how to clone for hindi language ? can i get procedure

**Comment by devinschumacher** [comment]
> how to clone for hindi language ? can i get procedure

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
